tests/lib/tools/cleanup-state

```diff
@@ -75,6 +75,12 @@ case "$action" in
 			# XXX: if there's a way to check if an unit exists but is stopped,
 			# use it here to avoid starting a non-existing unit.
 			systemctl --user start snapd.session-agent.socket || true
+
+                        # Do the same for root's D-Bus session bus
```

Comment:
Can you capture in this comment, why this is relevant to the rest of the system. I know that restarting the socket will close all connections. I'm not sure why we are doing this though.